target/linux/ath79/base-files/etc/uci-defaults/05_fix-compat-version

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (C) 2020 OpenWrt.org
+#
+
+. /lib/functions.sh
+
+case "$(board_name)" in
+	meraki,mr16)
+		uci set system.@system[0].compat_version="2.0"
+		uci commit system
+		;;
+esac
+
+exit 0
```

Comment:
as GitHub also marks here, this file has no newline at its end  (unlike e.g. the dts file in this PR)

Reply:
Fixed and pushed.